INSTALL_ASU.md

```diff
@@ -0,0 +1,62 @@
+
+
+After the installation of anaconda do the following
+```
+$ user=`whoami`
+$ env=epic
+ $ conda create -n ${env} python=2.7
+```
+It is advised to install the dependencies of LSL before attempting a `pip install`. While pip generally is good about grabbing package dependencies, the configuration of some packages requires dependencies to be installed before pip can check if the dependencies are installed.
+
+While not explicitly mentioned, lsl depends on `astropy` for fits I/O. Inclusion of astropy is not a mistake.
+
+To install dependencies for lsl excute the following lines. This may not be necessay depending on your environment but will help smooth the install.
+```
+$ conda activate ${env}
+$ conda install aipy scipy numpy fftw astropy healpy ephem pytz matplotlib
+$ pip install lsl
+```
+
+# Bifrost
+Until some pull requests are merged, we need to use a hybrid of two bifrost forks.
+```
+$ cd ~/src
+$ git clone https://github.com/ledatelescope/bifrost.git -b plugin-wrapper bifrost
+$ cd bifrost 
+$ git merge master  # merge in the lastest master to get work from James Kent's fork
+$ cp /data4/jdowell/CodeSafe/bifrost/src/proclog.cpp src/  # Make multi-user installation work
+```
+(The `proclog.ccp` file referenced above is also in this repository, `LWA_EPIC/config/ASU_proclog.cpp`, so you can copy from there in case anything ever happens to the one on `/data4/`.)
+
+Check `README.md` to install dependencies (do this in your epic env).
+Always check that the ctypesgen is the latest as mentioned in the README.md
+```
+$ pip install numpy contextlib2 pint git+https://github.com/olsonse/ctypesgen.git@9bd2d249aa4011c6383a10890ec6f203d7b7990f
+```
+
+Downgrade matplotlib
+```
+$ conda install matplotlib=2.2.3
+```
+
+Now we need to actually install bifrost. First set up configuration for intrepid. Copy the file `LWA_EPIC/config/ASU_user.mk` from this repository to your bifrost directory:
+```
+$ cp ~/src/LWA_EPIC/config/ASU_user.mk ~/bifrost/user.mk
+```
+
+Finally, install (note **this must be done on intrepid with your environment activated**):
+```
+$ make -j 32
+$ make install INSTALL_LIB_DIR="/home/${user}/src/anaconda/envs/${env}/lib" INSTALL_INC_DIR="/home/${user}/src/anaconda/envs/${env}/include" PYINSTALLFLAGS="--prefix=/home/${user}/src/anaconda/envs/${env}"
+```
+
+### Testing multiple branches
+If you want to install and test multiple bifrost branches, it is encouraged to use different conda environments. Check the environment variables of the install directory to properly link with the libraries.
+
+# EPIC
+https://github.com/epic-astronomy/epic
+
+TODO: Finish EPIC install instructions.
```

Comment:
I don't think this is needed anymore. But we probably do want to add an example showing how to run the script. And maybe even an example image to confirm you get the right thing.